data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/GetSyncCommitteeContribution.java

```diff
@@ -0,0 +1,123 @@
+import tech.pegasys.teku.api.response.v1.validator.GetSyncCommitteeContributionResponse;
+import tech.pegasys.teku.api.schema.altair.SyncCommitteeContribution;
+import tech.pegasys.teku.beaconrestapi.handlers.AbstractHandler;
+import tech.pegasys.teku.beaconrestapi.schema.BadRequest;
+import tech.pegasys.teku.infrastructure.async.SafeFuture;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.provider.JsonProvider;
+
+public class GetSyncCommitteeContribution extends AbstractHandler {
+  public static final String ROUTE = "/eth/v1/validator/sync_committee_contribution";
+
+  private final ValidatorDataProvider provider;
+
+  public GetSyncCommitteeContribution(
+      final DataProvider dataProvider, final JsonProvider jsonProvider) {
+    super(jsonProvider);
+    this.provider = dataProvider.getValidatorDataProvider();
+  }
+
+  @OpenApi(
+      path = ROUTE,
+      method = HttpMethod.GET,
+      summary = "Produce sync committee contribution",
+      tags = {TAG_EXPERIMENTAL},
+      queryParams = {
+        @OpenApiParam(
+            name = SLOT,
+            description =
+                "`uint64` The slot for which a sync committee contribution should be created.",
+            required = true),
+        @OpenApiParam(
+            name = SUBCOMMITTEE_INDEX,
+            description = "`uint64` The subcommittee index for which to produce the contribution.",
+            required = true),
+        @OpenApiParam(
+            name = BEACON_BLOCK_ROOT,
+            description = "`bytes32` The block root for which to produce the contribution.",
+            required = true)
+      },
+      description =
+          "Returns an unsigned sync committee contribution, for an validator to sign and post.",
+      responses = {
+        @OpenApiResponse(
+            status = RES_OK,
+            content = @OpenApiContent(from = GetSyncCommitteeContributionResponse.class)),
+        @OpenApiResponse(status = RES_BAD_REQUEST, description = "Invalid request syntax."),
+        @OpenApiResponse(status = RES_INTERNAL_ERROR, description = "Beacon node internal error.")
+      })
+  @Override
+  public void handle(@NotNull final Context ctx) throws Exception {
+    try {
+      final Map<String, List<String>> parameters = ctx.queryParamMap();
+      if (parameters.size() < 3) {
+        throw new IllegalArgumentException(
+            String.format(
+                "Please specify all of %s, %s and %s",
+                SLOT, SUBCOMMITTEE_INDEX, BEACON_BLOCK_ROOT));
+      }
+      final UInt64 slot = getParameterValueAsUInt64(parameters, SLOT);
+      final Bytes32 blockRoot = getParameterValueAsBytes32(parameters, BEACON_BLOCK_ROOT);
+      final int subcommitteeIndex = getParameterValueAsInt(parameters, SUBCOMMITTEE_INDEX);
+      if (subcommitteeIndex < 0) {
+        throw new IllegalArgumentException(
+            String.format("'%s' needs to be greater than or equal to 0.", SUBCOMMITTEE_INDEX));
+      }
```

Comment:
We should also check that `subcommitteeIndex < NetworkConstants.SYNC_COMMITTEE_SUBNET_COUNT` (yeah it's a constant in the spec and is not in the config file).